### ISSUE TITLE
Fritz dect 200 werden nicht erkannt 

### ISSUE THREAD

**Issue by Michaek76** [SeydX/homebridge-fritz-platform#295]
Hallo zusammen,
Ich nutze die Fritzplatform zusammen mit hoobs, alles zusammen wurde neu aufgesetzt und die Verbindung zur fritzbox steht, allerdings werden meine dect 200 Steckdosen nicht erkannt. 

Im Router ist alles konfiguriert, wie vorgegebene.

Unter configuration lässt sich smarthome auch nicht konfigurieren. 

Eine idee was falsch läuft?

ein neu installieren der fritzplatform hat auch keine Lösung gebracht.

**Comment by Michaek76** [comment]
Where can i get support?

**Comment by Michaek76** [comment]
Hello, still need support

**Comment by Michaek76** [comment]
What sould i do to get some support?

**Comment by BKrnk1337** [comment]
@Michaek76 could you post your config file? You should be able to add the devices there.

**Comment by Michaek76** [comment]
I tried to add the devices manually, but after I saved the config it was overwritten:


  "accessories": [],
    "platforms": [
        {
            "platform": "FritzPlatform",
            "name": "FritzPlatform",
            "options": {
                "polling": {
                    "timer": 15
                }
            },
            "devices": [
                {
                    "active": true,
                    "name": "FB7490",
                    "host": "192.168xxx.xxx",
                    "port": 49000,
                    "username": "xxxxx",
                    "password": "xxxxxx",
                    "ssl": true,
                    "tr064": true,
                    "igd": false,
                    "manufacturer": "Fritzbox",
                    "model": "7490",
                    "wifiUnits": 2,
                    "connection": "dsl",
                    "options": {
                        "wifi_2ghz": "switch",
                        "wifi_5ghz": "switch",
                        "wifi_guest": "switch",
                        "wps": "switch",
                        "dect": "switch",
                        "led": "switch",
                        "reconnect": "switch"
                    },
                    "master": true,
                    "oldFW": true
                }
            ],
            "debug": true,
            "error": true,
            "warn": true
        }
    ]
}

**Comment by BKrnk1337** [comment]
I would recommend you to delete your username and password. This is a big security breach posting this details online!

**Comment by Michaek76** [comment]
> I would recommend you to delete your username and password. This is a big security breach posting this details online!

Am so stupid :-( thx

**Comment by BKrnk1337** [comment]
But as I see your config is missing the smarthome part. You should go to the Homebridge GUI and follow the config for the plugin.

Mine looks like this:

```
  "smarthome" : [
    {
      "accType" : "thermostat",
      "active" : true,
      "ain" : "<your ain of the Device>",
      "battery" : true,
      "humidity" : false,
      "name" : "Thermostat KU",
      "temperature" : true,
      "window" : true
    },
    {
      "accType" : "thermostat",
      "active" : true,
      "ain" : "<your ain of the Device>",
      "battery" : true,
      "humidity" : false,
      "name" : "Thermostat BZ",
      "openWindow" : false,
      "temperature" : true,
      "window" : true
    },
    {
      "accType" : "thermostat",
      "active" : true,
      "ain" : "<your ain of the Device>",
      "battery" : true,
      "humidity" : false,
      "name" : "Thermostat SZ",
      "temperature" : true,
      "window" : true
    },
    {
      "accType" : "thermostat",
      "active" : true,
      "ain" : "<your ain of the Device>",
      "battery" : true,
      "humidity" : false,
      "name" : "Thermostat WZ",
      "temperature" : true,
      "window" : true
    },
    {
      "accType" : "switch",
      "active" : true,
      "ain" : "<your ain of the Device>",
      "energy" : true,
      "name" : "Steckdose Multimedia",
      "temperature" : true
    },
    {
      "accType" : "switch",
      "active" : true,
      "ain" : "<your ain of the Device>",
      "energy" : true,
      "name" : "Steckdose Schirmlampe",
      "temperature" : true
    }
  ],

```

But check out the example config.

https://github.com/SeydX/homebridge-fritz-platform/blob/master/example/example-config.json

As I mentioned before. The plugin supports a "guided" setup via the homebridge GUI (website)

Here some screenshots from the GUI:
![image](https://github.com/seydx/homebridge-fritz-platform/assets/27677748/69c6dcfa-efa5-4830-9b1e-0cfc371a07ab)

![image](https://github.com/seydx/homebridge-fritz-platform/assets/27677748/bd136bce-52fc-49c8-8f6f-9dd6abcfb3fb)

**Comment by Michaek76** [comment]
That's the problem. I can't activate the smarthome settings because there ist smarthome outlet.

<img width="847" alt="Bildschirmfoto 2023-10-07 um 13 00 32" src="https://github.com/seydx/homebridge-fritz-platform/assets/109075691/1d15d05c-ca92-4297-afa7-7cef5adaff1d">

I tried to add my dect 200 by editing the config file manually. But I can't see the changes, because after restarting the bridge, the text I added manually was deleted

**Comment by BKrnk1337** [comment]
Are you using homebridge? Can you show a screenshot like I did from the config? @seydx mentioned your device as "supported"

And I see that you are already in a more "deep" config. But in your settings you did not add it. You should do this before. 

And did you follow the setup steps correctly?

https://github.com/SeydX/homebridge-fritz-platform/blob/master/docs/Installation.md

**Comment by Michaek76** [comment]
Yes, when I used hoobs 3.0, everything was still running. Then I wanted to make the update to hoobs 4.0 and then nothing ran anymore, so I had to reinstall everything completely.

In Hoobs I have only the deep config under Plugins-> Fritz!Box platform Bridge 
<img width="787" alt="Bildschirmfoto 2023-10-07 um 13 22 05" src="https://github.com/seydx/homebridge-fritz-platform/assets/109075691/eb425d24-a07a-4c23-b960-35e3109bb72e">

my router settings are 
<img width="1176" alt="Bildschirmfoto 2023-10-07 um 13 17 55" src="https://github.com/seydx/homebridge-fritz-platform/assets/109075691/64705771-365a-4d4a-878d-5572fc0df2f5">

and 
<img width="1142" alt="Bildschirmfoto 2023-10-07 um 13 24 22" src="https://github.com/seydx/homebridge-fritz-platform/assets/109075691/7af49c19-b798-43e4-9c59-dbcae9c1a730">

**Comment by BKrnk1337** [comment]
I can't help you with hoobs. I am using homebridge
